src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Ios/CameraPreviewControlRenderer.cs

```diff
@@ -297,15 +297,11 @@ private void ApplyAspect()
 
             switch (cameraPreviewControl.Aspect)
             {
-                case Aspect.AspectFit:
+                case CameraPreviewAspect.AspectFit:
                     videoPreviewLayer.VideoGravity = AVLayerVideoGravity.ResizeAspect;
                     break;
 
-                case Aspect.AspectFill:
```

Comment:
Oh so we did have it in here - why are we removing it?

Reply:
Was in the description of the PR, Android has a list of supported preview sizes that are valid. If i'm not mistaken then if we do AspectFill, it means picking a larger size that could end up being cropped off and having a size that exceeds the bus' limitations that would lead to garbage images as per one of the comments in the Android camera samples around picking preview sizes. Though on second thought that is different than setting the size of the texture used for rendering) so will look at this again

Reply:
Will abandon PR for now and submit again later